### PR TITLE
SignIn module initial code checkin

### PR DESCRIPTION
**Pull request by NikhithaVS**

### CODE REVIEW

**Comment by simpleprahalad** [comment]
Please resolve the conflicts and re-raise PR. Thanks

**Comment by NikhithaVS** [comment]
Merge Conflicts resolved, request to review the code and merge.

File: quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import java.util.Base64;
+
+@RestController
+@RequestMapping("/")
+public class UserController {
+    @Autowired
+    private AuthenticationService authenticationService;
+
+    @RequestMapping(method= RequestMethod.POST,path="/user/signin",produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+    public ResponseEntity<SigninResponse> login(@RequestHeader("authorization") final String authorization) throws AuthenticationFailedException {
```
Comment (simpleprahalad) [inline quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java]: Can we rename login to signin ?

**Review by simpleprahalad — CHANGES_REQUESTED** [review]
Please do the change and commit it.

File: quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java (hunk 1 above)
Comment (NikhithaVS) [reply quora-api/src/main/java/com/upgrad/quora/api/controller/UserController.java]: Please check the latest commit with the changes updated

**Review by simpleprahalad — APPROVED** [review]
Changes look fine.

**Pull request merged**